addon-test-support/setup-global-a11y-hooks.ts

```diff
@@ -1,23 +1,75 @@
 import { _registerHook, HookUnregister } from '@ember/test-helpers';
-import { InvocationStrategy } from './types';
+import { InvocationStrategy, AuditFunction } from './types';
 import { getRunOptions } from './run-options';
 import a11yAudit from './audit';
 import { shouldForceAudit } from './should-force-audit';
 
+export interface GlobalA11yHookOptions {
+  helpers: HelperName[];
+}
+
+type HelperName =
+  | 'blur'
+  | 'click'
+  | 'doubleClick'
+  | 'fillIn'
+  | 'focus'
+  | 'render'
+  | 'scrollTo'
+  | 'select'
+  | 'tab'
+  | 'tap'
+  | 'triggerEvent'
+  | 'triggerKeyEvent'
+  | 'typeIn'
+  | 'visit';
+
 let _unregisterHooks: HookUnregister[] = [];
 
+export const DEFAULT_A11Y_TEST_HELPER_NAMES: HelperName[] = [
+  'visit',
+  'click',
+  'doubleClick',
+  'tap',
+];
+
 /**
  * Sets up a11yAudit calls using `@ember/test-helpers`' `_registerHook` API.
  *
  * @param shouldAudit Invocation strategy function that determines whether to run the audit helper or not.
  * @param audit Optional audit function used to run the audit. Allows for providing either a11yAudit, a11yAuditIf,
  *              or custom audit implementation.
  */
+export function setupGlobalA11yHooks(shouldAudit: InvocationStrategy): void;
+export function setupGlobalA11yHooks(
+  shouldAudit: InvocationStrategy,
+  audit: AuditFunction
+): void;
+export function setupGlobalA11yHooks(
+  shouldAudit: InvocationStrategy,
+  options: GlobalA11yHookOptions
+): void;
+export function setupGlobalA11yHooks(
+  shouldAudit: InvocationStrategy,
+  audit: AuditFunction,
+  options: GlobalA11yHookOptions
+): void;
 export function setupGlobalA11yHooks(
   shouldAudit: InvocationStrategy,
-  audit: (...args: any[]) => PromiseLike<void> = a11yAudit
-) {
-  ['visit', 'click', 'doubleClick', 'tap'].forEach((helperName) => {
+  auditOrOptions?: AuditFunction | GlobalA11yHookOptions,
+  options?: GlobalA11yHookOptions
+): void {
+  let audit: AuditFunction = a11yAudit;
+
+  if (typeof auditOrOptions === 'function') {
+    audit = auditOrOptions;
+  } else {
+    options = auditOrOptions;
+  }
+
+  let helpers = options?.helpers || DEFAULT_A11Y_TEST_HELPER_NAMES;
```

Comment:
If you declared a separate variable at the top of this scope and initialized it to `{}` you wouldn't need this null coalescing check.

```suggestion
  let audit: AuditFunction = a11yAudit;
  let opts: GlobalA11yHookOptions = {};

  if (typeof auditOrOptions === 'function') {
    audit = auditOrOptions;
  } else {
    opts = auditOrOptions;
  }

  let helpers = opts.helpers || DEFAULT_A11Y_TEST_HELPER_NAMES;
```

You'd also avoid having to turn off the `no-redeclare` rule in our eslint config.

Reply:
The linter noticed an issue with this. Will address